### ISSUE TITLE
TypeError: node.fsm[trigger] is not a function

### ISSUE THREAD

**Issue by colinl** [DeanCording/node-red-contrib-state-machine#1]
The following flow is generating the error "TypeError: node.fsm[trigger] is not a function" on clicking the Queue Not Empty trigger.  Probably I am doing something silly but can't see what.
```
21 Jan 13:52:57 - [info] Node-RED version: v0.17.5
21 Jan 13:52:57 - [info] Node.js  version: v8.9.4
21 Jan 13:52:57 - [info] Linux 4.9.35+ arm LE
```

[{"id":"18cf4e3e.85e29a","type":"state-machine","z":"31fa50e9.6a1698","name":"","triggerProperty":"topic","triggerPropertyType":"msg","stateProperty":"payload","statePropertyType":"msg","states":["queue_empty","queue_not_empty"],"transitions":[{"name":"queue_not_empty","from":"queue_empty","to":"queue_not_empty"},{"name":"queue_empty","from":"queue_not_empty","to":"queue_empty"}],"x":470.5,"y":690,"wires":[["a9dc4ecb.fe3858"]]},{"id":"48331391.a1d014","type":"change","z":"31fa50e9.6a1698","name":"queue empty","rules":[{"t":"set","p":"topic","pt":"msg","to":"queue_empty","tot":"str"}],"action":"","property":"","from":"","to":"","reg":false,"x":271.5,"y":654,"wires":[["18cf4e3e.85e29a"]]},{"id":"100ad065.b302d","type":"change","z":"31fa50e9.6a1698","name":"queue not empty","rules":[{"t":"set","p":"topic","pt":"msg","to":"queue_not_empty","tot":"str"}],"action":"","property":"","from":"","to":"","reg":false,"x":270,"y":727,"wires":[["18cf4e3e.85e29a"]]},{"id":"e26dc733.db43a8","type":"inject","z":"31fa50e9.6a1698","name":"","topic":"","payload":"","payloadType":"date","repeat":"","crontab":"","once":false,"x":98,"y":653,"wires":[["48331391.a1d014"]]},{"id":"1a1a830e.a4186d","type":"inject","z":"31fa50e9.6a1698","name":"","topic":"","payload":"","payloadType":"date","repeat":"","crontab":"","once":false,"x":97,"y":725,"wires":[["100ad065.b302d"]]},{"id":"a9dc4ecb.fe3858","type":"debug","z":"31fa50e9.6a1698","name":"state","active":true,"console":"false","complete":"payload","x":659,"y":690,"wires":[]}]

**Comment by colinl** [comment]
I think it is the underscores in the transitions that are doing it. I see that node.fsm has properties queueEmpty and queueNotEmpty rather than queue_empty and queue_not_empty.

**Comment by DeanCording** [comment]
Yes, it seems that the Java State Machine camelizes state and transition names but doesn't mention it in the documentation.  Should be a simple fix.

**Comment by DeanCording** [comment]
Fixed in Version 1.1.0
